scylla-server/src/utils/message-maps.utils.ts

```diff
@@ -1,8 +1,9 @@
 import { getAllDataTypes } from '../services/dataTypes.services';
 import { getAllDrivers } from '../services/driver.services';
 import { getAllSystems } from '../services/systems.services';
+import { getDataByDataTypeName } from '../services/data.services';
 
-export type ResponseFunction = (data?: JSON) => Promise<string>;
+export type ResponseFunction = (data: JSON) => Promise<string>;
```

Comment:
why are you making this nonoptional

Reply:
if its optional, functions that require data to run like #16 need to be able to deal with cases where they are supplied no data. we could throw an error or something but I felt like just removing the possibility of getting errors could be better. what do you think?